lexicons/app/bsky/feed/generator.json

```diff
@@ -30,6 +30,10 @@
             "accept": ["image/png", "image/jpeg"],
             "maxSize": 1000000
           },
+          "acceptsFeedback": { 
+            "type": "boolean",
+            "description": "Declaration that a feed accepts feedback from a client through app.bsky.feed.sendInteractions"
+          },
```

Comment:
I wonder if there's a way to unify the terminology around either "feedback" or "interactions."

Reply:
Yeah I was wondering that myself. I guess `acceptsInteractions` works just seemed a bit off 🤔

Reply:
thought about it & don't mind it so I changed it to `acceptsInteractions` 👍